diff --git a/node_modules/react-native-paper/src/styles/themes/v3/tokens.tsx b/node_modules/react-native-paper/src/styles/themes/v3/tokens.tsx
index d008944..b125359 100644
--- a/node_modules/react-native-paper/src/styles/themes/v3/tokens.tsx
+++ b/node_modules/react-native-paper/src/styles/themes/v3/tokens.tsx
@@ -123,69 +123,69 @@ const mediumType = {
 export const typescale = {
   displayLarge: {
     ...regularType,
-    lineHeight: 64,
+    // lineHeight: 64,
     fontSize: 57,
   },
   displayMedium: {
     ...regularType,
-    lineHeight: 52,
+    // lineHeight: 52,
     fontSize: 45,
   },
   displaySmall: {
     ...regularType,
-    lineHeight: 44,
+    // lineHeight: 44,
     fontSize: 36,
   },
 
   headlineLarge: {
     ...regularType,
-    lineHeight: 40,
+    // lineHeight: 40,
     fontSize: 32,
   },
   headlineMedium: {
     ...regularType,
-    lineHeight: 36,
+    // lineHeight: 36,
     fontSize: 28,
   },
   headlineSmall: {
     ...regularType,
-    lineHeight: 32,
+    // lineHeight: 32,
     fontSize: 24,
   },
 
   titleLarge: {
     ...regularType,
-    lineHeight: 28,
+    // lineHeight: 28,
     fontSize: 22,
   },
   titleMedium: {
     ...mediumType,
-    lineHeight: 24,
+    // lineHeight: 24,
     fontSize: 16,
   },
   titleSmall: {
     ...mediumType,
     letterSpacing: 0.1,
-    lineHeight: 20,
+    // lineHeight: 20,
     fontSize: 14,
   },
 
   labelLarge: {
     ...mediumType,
     letterSpacing: 0.1,
-    lineHeight: 20,
+    // lineHeight: 20,
     fontSize: 14,
   },
   labelMedium: {
     ...mediumType,
     letterSpacing: 0.5,
-    lineHeight: 16,
+    // lineHeight: 16,
     fontSize: 12,
   },
   labelSmall: {
     ...mediumType,
     letterSpacing: 0.5,
-    lineHeight: 16,
+    // lineHeight: 16,
     fontSize: 11,
   },
 
@@ -193,7 +193,7 @@ export const typescale = {
     ...mediumType,
     fontWeight: ref.typeface.weightRegular,
     fontFamily: ref.typeface.brandRegular,
-    lineHeight: 24,
+    // lineHeight: 24,
     fontSize: 16,
   },
   bodyMedium: {
@@ -201,7 +201,7 @@ export const typescale = {
     fontWeight: ref.typeface.weightRegular,
     fontFamily: ref.typeface.brandRegular,
     letterSpacing: 0.25,
-    lineHeight: 20,
+    // lineHeight: 20,
     fontSize: 14,
   },
   bodySmall: {
@@ -209,7 +209,7 @@ export const typescale = {
     fontWeight: ref.typeface.weightRegular,
     fontFamily: ref.typeface.brandRegular,
     letterSpacing: 0.4,
-    lineHeight: 16,
+    // lineHeight: 16,
     fontSize: 12,
   },
 
